### PR TITLE
Added preference panel to adjust hexagon radius and ripple size

### PR DESCRIPTION
**Pull request by vasutomar**
Added a preferences panel to adjust ripple size and hexagon radius

image

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,28 +1,104 @@
...
+                border-right: none;
+                border-bottom: 2px solid #020D78;
+            }
+          }
         }
+
+        .applybtn {
+            border: none;
+            height: 2rem;
+            width: 8rem;
```
Comment (moebrowne) [inline index.html]: When different fonts are loaded the text can overflow.
Screenshot from 2024-10-07 18-55-00

Please can you remove the fixed width, perhaps add some padding instead.

File: index.html [hunk 2]
```diff
@@ -1,28 +1,104 @@
...
-</head>
-<body>
+  </head>
+  <body>
+    <div class="hexplosion">
+      <div class="control-panel">
+        <div class="controls">
+            <div class="preference">
+              <label>Hexagon size</label>
+              <input id="radius" placeholder="Hexagon size" />
```
Comment (moebrowne) [inline index.html]: Could we add default values to the inputs, when it loads they are empty but it obviously still works.

**Review by moebrowne — CHANGES_REQUESTED** [review]
Thanks for the PR :rocket:

I've added a couple of comments of things which need adjusting

**Comment by vasutomar** [comment]
Hello, I have made the requested changes.
Thanks

**Comment by moebrowne** [comment]
Thanks!

**Pull request merged**